fix build warning for Ghidra 10.2 + JDK 17+

```
...\GhidrathonPlugin.java:53: warning: [removal] ProgramPlugin(PluginTool,boolean,boolean,boolean) in ProgramPlugin has been deprecated and marked for removal
                super(tool, true, true, true);
                ^
1 warning
```

--- a/src/main/java/ghidrathon/GhidrathonPlugin.java
+++ b/src/main/java/ghidrathon/GhidrathonPlugin.java
@@ -50,7 +50,7 @@ public class GhidrathonPlugin extends ProgramPlugin implements InterpreterConnec
 	
 	public GhidrathonPlugin(PluginTool tool) {
 		
-		super(tool, true, true, true);
+		super(tool);
 		
 	}
 

